Assets/NickelArcade/PrintScore.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
```

Comment:
add namespace enclosure
```suggestion
namespace Nickel{
```